keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py

```diff
@@ -102,7 +102,7 @@ def get_random_transformation(self, point_clouds, **kwargs):
         frustum_mask = tf.concat(frustum_mask, axis=0)
         random_point_mask = (
             self._random_generator.random_uniform(
-                [num_frames, num_points, 1], minval=0.0, maxval=1
+                [1, num_points, 1], minval=0.0, maxval=1
```

Comment:
any reason for this? Seems we're making it less generic

Reply:
If we need to make this, can you add a comment to why this is the case

Reply:
Similar to global drop point. We use the same point mask for all frames.

Reply:
Done.